src/client/src/admin.rs

```diff
@@ -6,9 +6,19 @@ use dataplane::core::Encoder;
 use dataplane::core::Decoder;
 use fluvio_sc_schema::objects::{Metadata, AllCreatableSpec};
 use fluvio_sc_schema::AdminRequest;
-use fluvio_socket::FlvSocketError;
-use fluvio_socket::AllMultiplexerSocket;
-use fluvio_future::native_tls::AllDomainConnector;
+
+#[cfg(not(target_arch = "wasm32"))]
+use fluvio_socket::AllMultiplexerSocket as FluvioMultiplexerSocket;
+
+#[cfg(not(target_arch = "wasm32"))]
+use fluvio_future::native_tls::AllDomainConnector as FluvioConnector;
+
+#[cfg(target_arch = "wasm32")]
+use crate::websocket::{
+    WebSocketConnector as FluvioConnector,
+    MultiplexerWebsocket as FluvioMultiplexerSocket,
+};
```

Comment:
Instead of directly using WebSocketConnector, It probably more flexible long term to move that to `FluvioSocket`.  In that case, Client would not needed to be exposed to WASM dependencies.